WeatherInfo/ViewControllers/Locations View Controller/Add Location View Controller/AddLocationViewController.swift

```diff
@@ -38,19 +38,21 @@ class AddLocationViewController: UIViewController {
         viewModel.locationsDidChange = { [unowned self] (locations) in
             self.tableView.reloadData()
         }
-        
+        self.activityIndicatorView.isHidden = true
```

Comment:
On main story board activity indicator's property, selected "Hide when Stopped'. No need to manually hide in code after this.

Reply:
What about the bug that you mentioned yesterday?

Reply:
This was introducing that bug. The activity indicator was already not hidden when the view loaded. Once we set this above property, we have to just start/Stop the animation to make the activity indicator view they way it should work.

Reply:
Then hidden is not the way. you should or should not be animating. 
As property says hide when stop animating. 
@lokeshub